# Parameters for the Gaussian Blur
gaussian_blur_size: 7
gaussian_blur_sigma: 2.0

# Parameters for the Hough transform
hough_accum_resolution: 2.0
min_circle_dist: 40.0
canny_edge_th: 150.0
hough_accum_th: 79.0
min_radius: 10
max_radius: 40
